chipy_org/urls.py

```diff
@@ -31,22 +32,22 @@
     path("job-board/", include("chipy_org.apps.job_board.urls")),
     path("talks/", include("chipy_org.apps.talks.urls")),
     path("tinymce/", include("tinymce.urls")),
-]
+] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
 
-# Would love a back tracking url resolver
 urlpatterns += [
     path("api/meetings/", MeetingListAPIView.as_view()),
     path("api/meetings/<int:meeting_id>/meetup/sync", MeetingMeetupSync.as_view()),
-]
+] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
 
 if settings.SERVE_MEDIA:
     urlpatterns += [
         path(
-            r"media/<path:path>",
+            "media/<path:path>",
             django.views.static.serve,
             {"document_root": settings.MEDIA_ROOT, "show_indexes": True},
         ),
     ]
     urlpatterns += staticfiles_urlpatterns()
+    urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
 
 handler404 = chipy_org.apps.main.views.custom_404
```

Comment:
Please don't delete the docker.env.sample.  Other devs will find that useful when setting up a local env.